### PR TITLE
Refactor `write_binary_recording` tests

### PR DESCRIPTION
**Pull request by h-mayorquin**
Refactoring tests for #1602. We should merge this one before.

### CODE REVIEW

File: src/spikeinterface/core/tests/test_core_tools.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import platform
-import pytest
 from pathlib import Path
 
-from spikeinterface.core import generate_recording
+import pytest
+import numpy as np
 
+from spikeinterface.core import generate_recording
```
Comment (alejoe91) [inline src/spikeinterface/core/tests/test_core_tools.py:7]:
```suggestion
```

not using this anymore right?

**Review by alejoe91 — APPROVED** [review]
A small comment Ramon!

File: src/spikeinterface/core/tests/test_core_tools.py (hunk 1 above)
Comment (h-mayorquin) [reply src/spikeinterface/core/tests/test_core_tools.py]: Thanks

**Comment by h-mayorquin** [comment]
> A small comment Ramon!

Turns out it was still needed. But now it is not anymore.

**Pull request merged**